src/main/java/com/fundation/logic/useful/ByteConvert.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) 2019 Jalasoft.
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+package com.fundation.logic.useful;
```

Comment:
I think this class should be placed in "common" package

Reply:
Ok that was correct.